app/views/hq/orphans/_form.html.erb

```diff
@@ -0,0 +1,157 @@
+<%= form_for [:hq, @orphan] do |f| %>
+  <%= render 'shared/errors', object: f.object %>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h3 class="panel-title">Orphan Details</h3>
+    </div>
+    <div class="panel-body">
+      <div class="form-group">
+        <%= f.label :name %>
+        <%= f.text_field :name, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :date_of_birth %>
+        <%= f.text_field :date_of_birth, class: "form-control date-picker", value: @orphan.date_of_birth %>
+      </div>
+      <div class="form-group">
+        <%= f.label :gender %>
+        <%= f.select :gender,
+          Settings.lookup.gender, {}, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :health_status %>
+        <%= f.text_field :health_status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :schooling_status %>
+        <%= f.text_field :schooling_status, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_status_id %>
+        <%= f.collection_select :orphan_status_id,
+          @orphan_statuses, :id, :name, {}, 
+          {class: "form-control"} %>
+      </div>
+      <div class="form-group">
+        <%= f.label :orphan_sponsorship_status_id %>
+        <%= f.collection_select :orphan_sponsorship_status_id,
+          @orphan_sponsorship_statuses, :id, :name, {}, 
+          {class: "form-control", disabled: !f.object.new_record?} %>
```

Comment:
Sponsorship status will not be manually set under any circumstances. I realize that it's a moot point since the form is not used for new records, but there simply is no need for the conditional, the field should be always disabled.